libs/s25main/ingameWindows/iwMinimap.cpp

```diff
@@ -41,7 +44,16 @@ iwMinimap::iwMinimap(IngameMinimap& minimap, GameWorldView& gwv)
     // Fenster vergrößern/verkleinern
     AddImageButton(4, DrawPoint(0, 0), BUTTON_SIZE, TextureColor::Grey, LOADER.GetImageN("io", 109));
 
-    Resize(GetSize());
+    Resize(extended ? MINIMAP_SIZE_BIG : MINIMAP_SIZE);
+}
+
+iwMinimap::~iwMinimap()
+{
+    // Possibly save our old position
+    if(helpers::contains(persistentWindows, GetID()))
+    {
+        SETTINGS.windows.persistentSettings[GetID()].option = (unsigned)extended;
```

Comment:
```suggestion
        SETTINGS.windows.persistentSettings[GetID()].option = static_cast<unsigned>(extended);
```

Reply:
I now changed the minimap extension to an explicit bool option in the ingame section of the SETTINGS instead, which makes this cast unneccesary